### DISCUSSION TITLE
How to restrict state at component level?

### DISCUSSION THREAD

**Discussion by krishnaeverestengineering** [GrapesJS/grapesjs#3929] [Ideas]
I want to disable hover state for particular components. How to do that? May be add a function hideStates([]) in state api?

**Comment by artf** [comment]
The state list is controlled by [Selector Manager](https://grapesjs.com/docs/api/selector_manager.html), you have the methods and events to do any type of customization (eg. changing the list based on the component type).

**Reply by manyang-megantoro** [reply]
How to remove all selector? So I can make user force to just get the block as selected not all component of the block.
